pkg/syncer/cnsoperator/controller/csinodetopology/csinodetopology_controller.go

```diff
@@ -361,6 +361,7 @@ func (r *ReconcileCSINodeTopology) reconcileForGuest(ctx context.Context, reques
 		// Error reading the object - return with err.
 		return reconcile.Result{}, err
 	}
+	// TODO: If the CR status is already at Success, do not reconcile further.
```

Comment:
Can we mention this is required when Node VM moves from one AZ to another AZ, otherwise there is no functional impact.

Reply:
Done